### ISSUE TITLE
[4.x] New replicator sets get inserted when using bard inside a replicator set

### ISSUE THREAD

**Issue by flolanger** [statamic/cms#7998]
### Bug description

Video of the Bug: https://cdn.discordapp.com/attachments/1075829281298587658/1100416349743886447/Bildschirmaufnahme_2023-04-25_um_15.38.49.mov

It seems like when using bard in a replicator and using enter key to create some new lines, new replicator items get inserted (see video).

### How to reproduce

1. Have a Bard field inside the first replicator group
2. Create another group inside the same replicator with any field inside
3. Create a new entry
4. Add the bard field from the first replicator group
5. Hit the enter key multiple times inside the bard field
6. -> New replicator sets appear

### Environment

```yaml
Environment
Application Name: Laravel
Laravel Version: 10.8.0
PHP Version: 8.2.4
Composer Version: 2.5.4
Environment: local
Debug Mode: ENABLED
URL: localhost
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 0
Antlers: runtime
Stache Watcher: Enabled
Static Caching: Disabled
Version: 4.0.0-alpha.4 Solo
```

### Installation

Fresh statamic/statamic site via CLI

### Antlers Parser

runtime (new)

**Comment by jasonvarga** [comment]
Yup, it was the group which is what I was missing. Sorry, and thanks!

**Comment by jasonvarga** [comment]
Fixed in #8004
